components/organisms/TeamSection/TeamSection.tsx

```diff
@@ -0,0 +1,51 @@
+import Card from "@/components/atoms/Card";
+import alexis from "@/public/Alexis.png";
+import shedrack from "@/public/shedrack.png";
+import Image from "next/image";
+import Link from "next/link";
+
+const teamData = [
+  {
+    userName: "Shedrack Akintayo",
+    position: "Founder",
+    imgUrl: "/shedrack.png",
+  },
+  {
+    userName: "Seunbayo Adebayo",
+    position: "Advocacy",
+    imgUrl: "",
+  },
+  {
+    userName: "Amarachi Iheanacho",
+    position: "Technical Writer",
+    imgUrl: "",
+  },
+  {
+    userName: "Damilola Ezekiel",
+    position: "Technical Writer",
+    imgUrl: "",
+  },
+  {
+    userName: "Alexis Abakasanga",
+    position: "Designer",
+    imgUrl: "/alexis.png",
+  },
+];
+export function TeamSection() {
+  return (
+    <section id="teams" className="bg-[#E4F4E9] p-8 flex flex-col gap-8">
```

Comment:
We already have the background color in the project as a tailwind config, you can access using `bg-primary-light`